build/DependencyVersions.props

```diff
@@ -1,34 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_SharedFrameworkVersion>2.0.3</CLI_SharedFrameworkVersion>
-    <CLI_MSBuild_Version>15.4.8</CLI_MSBuild_Version>
-    <CLI_Roslyn_Version>2.3.2-beta1-61921-05</CLI_Roslyn_Version>
+    <MicrosoftNETCoreAppPackageVersion>2.0.3</MicrosoftNETCoreAppPackageVersion>
```

Comment:
Shouldn't these be set conditionally? How will the downloaded version file overwrite this?

Reply:
The conditionals are in the downloads version prop file. Example: 
`<SystemMemoryPackageVersion Condition="$(SystemMemoryPackageVersion.Contains('-'))">4.5.0-preview1-25930-01</SystemMemoryPackageVersion>`

Reply:
I don't follow then. I thought the goal of this file was that if it was passed in, then its values would win. The way it is coded up right now, if will only win if the original value of the dependency does not contain a suffix in its version. Is this really the correct semantics for this?

Reply:
@weshaggard @dagood Did we decide to remove the conditionals from the imported dependency props and spread them into the repo-specific version props?  I think we did, to allow for more flexibility on the repo side.

Reply:
Yes we are removing the conditions in the generated package list, so repo's need to correctly condition or name their properties.

Reply:
Wasn't the idea to import the downloaded file after our property group? I don't get why we need conditions.